### PR TITLE
 Extract comments from html files along with widl when possible

### PR DESCRIPTION
**Pull request by mhegazy**
Some of the input files have a non-normative descriptions of the interface meant for end-user use. This change extracts these comments and ads them to the generated files.

### CODE REVIEW

**Comment by mhegazy** [comment]
@saschanaz can you please review this change.

File: tsconfig.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "compilerOptions": {
         "target": "es6",
```
Comment (saschanaz) [inline tsconfig.json]: Will there is a trouble if we just set target: es2016?

**Comment by mhegazy** [comment]
Looks like not all of the widl files have the `domintro` section. i will need to enhance this logic a bit.

**Comment by saschanaz** [comment]
Is there a blocking problem here?

**Comment by mhegazy** [comment]
Some legal issues I need to sort out. Need to figure out if the license allows such use and what attribution is needed.

**Pull request merged**